firestore/src/main/java/com/firebase/ui/firestore/paging/PageKey.java

```diff
@@ -20,7 +21,8 @@ public PageKey(@Nullable DocumentSnapshot startAfter, @Nullable DocumentSnapshot
         mEndBefore = endBefore;
     }
 
-    public Query getPageQuery(Query baseQuery, int size) {
+    @NonNull
```

Comment:
Please update the PR title to reflect all the nullability stuff.

Reply:
Will be moved into another PR